rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -942,10 +955,40 @@ protected void doDUP() {
         program.step();
     }
 
+    protected void doDUPN() {
+        spendOpCodeGas();
+        // EXECUTION PHASE
+        program.step();
+
+        int n = stack.pop().intValueCheck() + 1;
+
+        program.verifyStackSize(n);
+        program.verifyStackOverflow(n, n + 1);
```

Comment:
Stack is checked at steps(), and checking implementaitons it adds nothing doing this here. We can add 0, 1 for stack overflow.